docker/scripts/osie.sh

```diff
@@ -130,11 +130,18 @@ if ! [[ -f /statedir/disks-partioned-image-extracted ]]; then
 	mkdir $assetdir
 	echo -e "${GREEN}#### Fetching image (and more) via git ${NC}"
 
-	# config hosts entry so git-lfs assets are pulled through our image cache
-	githost="images.packet.net"
-	images_ip=$(getent hosts $githost | awk '{print $1}')
+	# config hosts entry so git-lfs assets from github and our github-mirror are
+	# pulled through our image cache
+	githost="github-mirror.packet.net"
+	if ! github_mirror_check; then
```

Comment:
will this fail when run in pre-install mode and dev has the non-routable 172/16 ip? See call to  `ensure_reachable` on line 164.

Reply:
Good catch. I've updated the github_mirror_check() function to include some of that logic and handle that situation now. I've also noticed I wasn't using colors in my warning messages and have updated those accordingly.

Reply:
I don't feel too great about extracting some of the ensure_reachable logic into github_mirror_check, why not do:
```sh
if ! (ensure_reachable github-mirror.packet.net && github_mirror_check) ; then
...
```